<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tratar Chamados</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_geral">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_menu">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>221</width>
       <height>361</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 255);
border-radius: 20px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_Concluir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(0, 47, 0);
</string>
      </property>
      <property name="text">
       <string> Concluir um chamado</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../.designer/src/verificado.png</normaloff>../../../../../.designer/src/verificado.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Select">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
</string>
      </property>
      <property name="text">
       <string> Selecionar Chamado</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../.designer/src/selecione.png</normaloff>../../../../../.designer/src/selecione.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Voltar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string> Voltar</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../.designer/src/sair.png</normaloff>../../../../../.designer/src/sair.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Limpar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(171, 141, 23);</string>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../.designer/src/limpar-limpo.png</normaloff>../../../../../.designer/src/limpar-limpo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>691</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../../.designer/src/lista-de-desejos.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Lista de Chamados</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_select">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>471</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(91, 91, 91);
border-radius: 20px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Digite o ID do chamado a ser Selecionado:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_Select">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000; 
background-color: rgb(207, 207, 207);
border-width: 2px; border-radius: 10px; border-color: #000;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>421</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_SelectCmd">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
</string>
      </property>
      <property name="text">
       <string> Selecionar Chamado</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../.designer/src/selecione.png</normaloff>../../../../../.designer/src/selecione.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 195, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetSelect">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>411</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="res_Select">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_conclui">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>470</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(91, 91, 91);
border-radius: 20px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Digite o ID do chamado a ser Finalizado:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_Conclui">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000; 
background-color: rgb(207, 207, 207);
border-width: 2px; border-radius: 10px; border-color: #000;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>421</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 195, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ConcluirCmd">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(0, 47, 0);</string>
      </property>
      <property name="text">
       <string> Concluir Chamado</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../.designer/src/verificado.png</normaloff>../../../../../.designer/src/verificado.png</iconset>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetConcluir">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>411</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="res_Conclui">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_off">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>471</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
background-color: rgb(130, 130, 130);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
